.gitignore

```diff
@@ -71,7 +71,7 @@ typings/
 # dotenv environment variables file
 .env
 .env.test
-
+.env-prod
```

Comment:
Bruker vi `.env.test` og `.env.prod` eller `.env-test` og `.env-prod`, eller en god kombo?

Bare så vi ikke får feil fordi vi blander;)

Reply:
kan vel ha en god kombo i guess? 🤷🏼‍♂️ så lenge det gir mening for den som holder på har det vel ikke så mye å si

Reply:
høres bra ut. har egt ingen formening, lurte bare på om det var med vilje eller ikke